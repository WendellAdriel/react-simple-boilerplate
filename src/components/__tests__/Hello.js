import React from 'react'
import renderer from 'react-test-renderer'
import Hello from 'components/Hello'

describe('Hello (Snapshot)', () => {
  it('Hello renders hello world', () => {
    const component = renderer.create(<Hello />)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })
})
