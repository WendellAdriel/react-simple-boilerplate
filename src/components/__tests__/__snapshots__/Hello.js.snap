// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello (Snapshot) Hello renders hello world 1`] = `
<h1>
  Hello, world!
</h1>
`;
